dev-libs/rapidjson/rapidjson-1.1.0.recipe

```diff
@@ -0,0 +1,66 @@
+SUMMARY="A fast JSON parser and generator for C++ with both SAX/DOM style API"
+DESCRIPTION="RapidJSON is a JSON parser and generator for C++. \
+It was inspired by RapidXml. RapidJSON is small but complete. \
+RapidJSON is fast. Its performance can be comparable to strlen().\
+RapidJSON is self-contained and header-only. RapidJSON is memory-friendly. \
+RapidJSON is Unicode-friendly"
+HOMEPAGE="https://github.com/Tencent/rapidjson"
+COPYRIGHT="2015 THL A29 Limited, a Tencent company
+	2015 Milo Yip"
+LICENSE="MIT
+	JSON"
+REVISION="1"
+SOURCE_URI="https://github.com/Tencent/rapidjson/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="bf7ced29704a1e696fbccf2a2b4ea068e7774fa37f6d7dd4039d0787f8bed98e"
+SOURCE_FILENAME="rapidjson-$portVersion.tar.gz"
+PATCHES="rapidjson-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	rapidjson$secondaryArchSuffix = $portVersion
+	lib:rapidjson$secondaryArchSuffix = $portVersion compat >= $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	rapidjson${secondaryArchSuffix}_devel = $portVersion compat >= $portVersion
+	devel:rapidjson$secondaryArchSuffix = $portVersion compat >= $portVersion
+	"
+REQUIRES_devel="
+	rapidjson$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:c++$secondaryArchSuffix
+	cmd:cmake
+	cmd:make
+	"
+
+BUILD()
+{
+	mkdir -p build
+	cd build
+	cmake ..
+	make
+}
+
+INSTALL()
+{
+	mkdir -p $libDir/pkgconfig
```

Comment:
Did you forgot to run the actual installation process?

Reply:
Sorry for the late reply. Rapidjson is a header only library. There is no lib files actually. Files copied to `$libDir/pkgconfig` are for cmake.